resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml.bak

```diff
@@ -0,0 +1,1387 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
This garbage can be removed. This is a left-over from sed operation in `install-solr.sh` which shouldn't be committed in a first place. It's already in a proper form in `resources/platformsh/ibexa-commerce/3.3/.platform/configsets/solr6/conf/solrconfig.xml`